code/game/machinery/stasis.dm

```diff
@@ -19,17 +18,32 @@
 	var/obj/effect/overlay/vis/mattress_on
 	var/obj/machinery/computer/operating/op_computer
 
+// dir check for buckle_lying state
+/obj/machinery/stasis/Initialize()
+	switch(dir)
+		if(WEST, NORTH)
+			buckle_lying = 270
+		if(EAST, SOUTH)
+			buckle_lying = 90
+	return ..()
+
 /obj/machinery/stasis/Initialize(mapload)
 	..()
 	return INITIALIZE_HINT_LATELOAD
 
 /obj/machinery/stasis/LateInitialize()
 	. = ..()
 	initial_link()
+	for(var/direction in GLOB.alldirs)
+		op_computer = locate(/obj/machinery/computer/operating) in get_step(src, direction)
+		if(op_computer)
+			op_computer.sbed = src
+			src.op_computer = op_computer
```

Comment:
I think this is reverting a change made in another PR?

Reply:
Thanks for the spot, should be resolved